hs_core/hydroshare/hs_bagit.py

```diff
@@ -43,36 +36,34 @@ def create_bag(resource):
     """
     from . import utils as hs_core_utils
     DATE_FORMAT = "YYYY-MM-DDThh:mm:ssTZD"
+    istorage=IrodsStorage()
 
     dest_prefix = getattr(settings, 'BAGIT_TEMP_LOCATION', '/tmp/hydroshare/')
     bagit_path = os.path.join(dest_prefix, resource.short_id, arrow.get(resource.updated).format("YYYY.MM.DD.HH.mm.ss"))
-    visualization_path = os.path.join(bagit_path, 'visualization')
-    contents_path = os.path.join(bagit_path, 'contents')
 
-    for d in (dest_prefix, bagit_path, visualization_path, contents_path):
+    for d in (dest_prefix, bagit_path):
         try:
             os.makedirs(d)
```

Comment:
This method can throws OSError with two codes: 1) errno.EEXIST, which you handle implicitly by calling shutil.rmtree(); and 2) errno.EACCES, which this code does not handle.  Ideally, you should handle both cases, and in the case of errno.EACCES, we should write a message to a log.  Later, we can configure a critical error log handler that e-mails entries somewhere such as an administrator e-mail list, or to a monitoring system.